lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -0,0 +1,79 @@
+module Fastlane
+  module Actions
+    class AndroidDownloadFileByVersionAction < Action
+      def self.run(params)
+        require_relative '../../helper/android/android_localize_helper.rb'
+        require_relative '../../helper/github_helper'
+
+        version = Fastlane::Helper::Android::LocalizeHelper.get_library_version_from_gradle_config(import_key: params[:import_key])
+        UI.user_error!("Can't find any reference for key #{params[:import_key]}") if version.nil?
+        UI.message "Downloading #{params[:file_path]} from #{params[:repository]} at version #{version} to #{params[:download_folder]}"
+
+        Fastlane::Helper::GithubHelper.download_file_from_release(
+          repository: params[:repository],
+          release: "#{params[:github_release_prefix]}#{version}",
+          file_path: params[:file_path],
+          download_folder: params[:download_folder]
+        )
```

Comment:
```suggestion
        Fastlane::Helper::GithubHelper.download_file_from_tag(
          repository: params[:repository],
          tag: "#{params[:github_release_prefix]}#{version}",
          file_path: params[:file_path],
          download_folder: params[:download_folder]
        )
```

Reply:
Good catch! TIL that in VSCode, global search uses the actual content of the files opened in the editor, even if they are not saved :-) 
I found this and actually changed it, but I forgot to save the file and when I double checked using the search function, this didn't show up 🤦

Reply:
Yeah, that's one of the reason why I missed a couple of things in the first place when I did my refactoring of release-toolkit – as you already saw in this PR with [this](https://github.com/wordpress-mobile/release-toolkit/pull/222#discussion_r589602025) 😅 

_Switching to RubyMine was a big deal and help in that regard (and I'll probably never be using VSCode ever again to edit/refactor Ruby code from now on, as RubyMine makes these kind of things less error-prone 😉)_